app/config/sources.inc.php

```diff
@@ -446,13 +448,14 @@
     'fx_android/whatsnew/android_45.lang'      => [ 'obsolete' => ['all'] ],
     'fx_android/whatsnew/android_46.lang'      => [ 'obsolete' => ['all'] ],
     'fx_android/whatsnew/android_47.lang'      => [ 'obsolete' => ['all'] ],
-    'fx_android/whatsnew/android_48.lang'      => [ 'obsolete' => ['all'] ],
```

Comment:
Why are you removing 48 here? we should actually mark 49 and 50 here as obsolete (51 won't be for a few days more).